storage/src/vespa/storage/storageserver/mergethrottler.cpp

```diff
@@ -1217,6 +1233,17 @@ MergeThrottler::markActiveMergesAsAborted(uint32_t minimumStateVersion)
     }
 }
 
+void MergeThrottler::apply_timed_backpressure() {
+    vespalib::LockGuard lock(_stateLock);
+    _throttle_until_time = _component.getClock().getMonotonicTime() + _backpressure_duration;
+    // TODO add exponential backoff if we deem it necessary
```

Comment:
What's the argument for preferring exponential backoff? Once the backpressure duration has passed, if the provider still suffers from resource exhaustion, backpressure will just be applied again. Exponential backoff could mean that backpressure is applied for much longer than necessary.

Reply:
It's not a preference as per se, mostly a comment of "if we need it, this is where we'll shove the functionality for it". But I'd agree it's premature to keep it in there now. Will remove it.